NOTICE

```diff
@@ -9,7 +9,7 @@ Version 2.0, an Apache-compatible license.
 * For a copy of the Apache License Version 2.0, please see LICENSE
   as included in this repository's top-level directory.
 
-* For a copy of the Neural Magic DeepSparse Community License, please see LICENSE-NEURALMAGIC
+* For a copy of the Neural Magic DeepSparse Community License, please see LICENSE
```

Comment:
is this redundant/conflicting with the previous bullet?

Reply:
Ah, yes good catch there. Let me update.